airflow/www_rbac/static/package.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
Something about having all these type of files inside `static/` feels wrong to me -- seeing that directory I would expect that just to contain static files to be served by a webserver - not config files such as this

Reply:
This file holds all dependencies (JS packages, and their versions) and the commands to build those packages. I was thinking to place this in `www_rbac` directory, but these are purely frontend (JS & CSS) dependent files, and usually placed under static or assets directories.

Reply:
They are not static asset files themselves though. Seeing this here I'd expect to be able to hit `http://my-airflow/static/package.json` or similar. Not having it under static removes that possible confusion entirely.